doc/source/tutorials/index.rst

```diff
@@ -1,11 +1,19 @@
 Tutorials
 =========
-You can find some examples on how to use martinize 2 in the martinize-examples
+This page contains several examples of how martinize2 can be used to convert
+proteins from atomistic to a martini3 representation.
+
+You can find further examples on how to use martinize2 in the martinize-examples
 repository: https://github.com/marrink-lab/martinize-examples
 
 .. Organize the tutorials in directories, so it's easier to keep their files
     together
 
 .. toctree::
+    basic_usage
+    elastic_networks
+    go_models
+    mutations_and_modifications
+    water_biasing
```

Comment:
I think it makes more sense to swap mutations_and_modifications and water_biasing